source/algorithms.tex

```diff
@@ -667,6 +667,11 @@
                   ForwardIterator first, ForwardIterator last, Function f);
 
   namespace ranges {
+    template<input_iterator I, class Proj = identity,
```

Comment:
Someone will need to apply P2106R0's addition of `for_each_n_result`. Not sure if that should happen here, in #3774, or separately.

Reply:
@burblebee , please do that in a separate commit in this pull request. Thanks.

Reply:
done, though I wasn't sure how the commit message should be worded

Reply:
Thanks for doing this. I'm going to move these edits to the LWG 23 branch as part of merging; I think they make more sense there since that was the motion that directed us to make the edits. Putting them here prior to merge is fine, though -- I expect it saved you from needing to branch LWG23 off LWG22.

FYI: in future, it'd be useful to leave a comment on the pull request when things like this are necessary, so I understand what's going on when I come to merge. (I read through the "Conversation" tab on the pull request, but not through resolved comments on the patch itself.)